src/code/z_sub_s.c

```diff
@@ -36,9 +36,120 @@ EnDoor* SubS_FindDoor(GlobalContext* globalCtx, s32 unk_1A5) {
     return door;
 }
 
-#pragma GLOBAL_ASM("asm/non_matchings/code/z_sub_s/func_8013A860.s")
+Gfx* SubS_DrawTransformFlexLimb(GlobalContext* globalCtx, s32 limbIndex, void** skeleton, Vec3s* jointTable,
+                                OverrideLimbDraw overrideLimbDraw, PostLimbDraw postLimbDraw,
+                                TransformLimbDraw transformLimbDraw, Actor* actor, Mtx** mtx, Gfx* gfx) {
+    StandardLimb* limb;
+    Gfx* newDList;
+    Gfx* limbDList;
+    Vec3f pos;
+    Vec3s rot;
+
+    Matrix_StatePush();
+    limb = (StandardLimb*)Lib_SegmentedToVirtual(skeleton[limbIndex]);
+    limbIndex++;
+    rot = jointTable[limbIndex];
+    pos.x = limb->jointPos.x;
+    pos.y = limb->jointPos.y;
+    pos.z = limb->jointPos.z;
+    newDList = limbDList = limb->dList;
+
+    if ((overrideLimbDraw == NULL) || !overrideLimbDraw(globalCtx, limbIndex, &newDList, &pos, &rot, actor, &gfx)) {
+        Matrix_JointPosition(&pos, &rot);
+        Matrix_StatePush();
+
+        //! @bug Does not check transformLimbDraw is not NULL before calling it.
+        transformLimbDraw(globalCtx, limbIndex, actor, &gfx);
```

Comment:
Interesting. I'm not sure if this should be noted as a bug tho, since there are lot of places where the game expects a non NULL pointer.

I think this is something important and should be noted explicitly, but I'm not sure if this is the better way to do it.